### PR TITLE
For #4625: Delete browsing data string unclear 

### PR DESCRIPTION
**Pull request by Pheonix73**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/fenix/blob/master/CHANGELOG.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -623,7 +623,8 @@
     <string name="preferences_delete_browsing_data_button">Delete browsing data</string>
 
     <!-- Dialog message to the user asking to delete browsing data. -->
-    <string name="delete_browsing_data_prompt_message">This will delete all of your browsing data.</string>
+    <string name="delete_browsing_data_prompt_message">This will delete the data you selected.</string>
```
Comment (pocmo) [inline app/src/main/res/values/strings.xml]: CC @mheubusch

Comment (colintheshots) [reply app/src/main/res/values/strings.xml]: We also need to rename strings when we change them as we don't have another process yet for updating strings across every translation. Let's wait for strings approval though.

Comment (Pheonix73) [reply app/src/main/res/values/strings.xml]: So should I close this PR or wait?

Comment (sblatz) [reply app/src/main/res/values/strings.xml]: @mheubusch has already approved these strings: https://github.com/mozilla-mobile/fenix/issues/4625#issuecomment-519736148

@Pheonix73 are you just able to update the 
` <string name="delete_browsing_data_prompt_message">` 
to say something like:  `<string name="delete_browsing_data_dialog_message">`? That will make it so the string is identified as new and can be translated :)

**Comment by boek** [comment]
@Pheonix73 

https://github.com/mozilla-mobile/fenix/pull/4703#discussion_r313977193

Is this still something you want to do? :)

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Pheonix73) [reply app/src/main/res/values/strings.xml]: @boek definetly. i will send a PR tomorrow.

**Comment by boek** [comment]
Ah sorry, I didn't see your comment on the issue! This was fixed with https://github.com/mozilla-mobile/fenix/commit/1b2757be981f5287551472d9dd47e815ac29b9b8

**Pull request closed without merging**